app/models/spree/user_decorator.rb

```diff
@@ -8,6 +8,14 @@ def supplier?
     supplier.present?
   end
 
+  def supplier_admin?
```

Comment:
should this method accept an argument (an instance of supplier)? i.e., should a user be a supplier_admin _for_ a particular supplier?
oh wait... a user belongs to a single supplier... never mind.